web/frontend/angular/src/app/main-pages/register/register.component.ts

```diff
@@ -36,7 +38,20 @@ export class RegisterComponent implements OnInit {
         if (this.authForm.invalid) {
             return
         }
-        this.authService.register(this.authForm.value)
-        this.router.navigateByUrl("homepage")
+        this.authService.register(this.authForm.value).subscribe(
+            (res: any) => {
+                console.log(res)
+                this.router.navigateByUrl("homepage")
+            },
+            (error) => {
+                console.log(error)
```

Comment:
Remove this log. If needed, this data can be seen in the developer console by inspecting the network request in the network tab.